### DISCUSSION TITLE
Unable to import roles.ExecutableOptionRole

### DISCUSSION THREAD

**Discussion by riwu** [sqlalchemy/sqlalchemy#10892] [Usage Questions]
```
from sqlalchemy.sql.roles import ExecutableOptionRole

def fn(option: ExecutableOptionRole):
    select(...).options(option)
```
https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/roles.py#L61

Why am I not able to import `ExecutableOptionRole`?      It gives an error `unknown import symbol`.

Or is there a better way to add typing for `option`?

**Comment by zzzeek** [comment]
hi -

what's "it" here?  is this your type checker ?   make sure there's no stubs installed

**Reply by riwu** [reply]
Yes, both pylance and mypy. 

Do you mean other libraries could contain stubs that interfere with the sqlalchemy import? How does that work since the import specifies the module path `sqlalchemy.sql.roles`?
